clients/src/main/java/org/apache/kafka/clients/admin/UpdateFeaturesOptions.java

```diff
@@ -26,4 +26,14 @@
  */
 @InterfaceStability.Evolving
 public class UpdateFeaturesOptions extends AbstractOptions<UpdateFeaturesOptions> {
+    private boolean validateOnly = false;
+
+    public boolean validateOnly() {
+        return validateOnly;
+    }
+
+    public UpdateFeaturesOptions dryRun(boolean dryRun) {
```

Comment:
looks like this is a lingering reference to `dryRun`. it should be `validateOnly` like the others...

Reply:
I wasn't sure if we needed to leave it as-is for compatibility reasons.

Reply:
Changing the name of the parameter doesn't change the binary compatibility, so it should be fine. One advantage of Java not having named parameters, I guess.

Reply:
Looking at this again, I can see that I forgot that it isn't just the parameter name, but also the method name.

I guess then let's just deprecate this one and add the validateOnly one